go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/operator-framework/operator-marketplace
 
 go 1.23.0
 
-toolchain go1.23.4
+toolchain go1.24.1
```

Comment:
We cannot do this change I think.
We need to do this one manually

Reply:
Closed in favor of: https://github.com/operator-framework/operator-marketplace/pull/611

@anik120 this one we will need to do manually